README.md

```diff
@@ -91,7 +91,7 @@ However some of the accelerator back-end implementations require different boost
 When an accelerator back-end using *Boost.Fiber* is enabled, `boost-fiber` and all of its dependencies are required to be built in C++11 mode `./b2 cxxflags="-std=c++11"`.
```

Comment:
Is C++11 fine here, or should it also become C++14?

Reply:
C++11 should be enough. C++14 also works.